site/content/contribute/webapp/build-component.md

```diff
@@ -170,116 +170,3 @@ export default class ItemList extends React.PureComponent {
     }
 }
 ```
```

Comment:
Maybe add a link to the testing page at the bottom here after a divider?

e.g.


```
---
To test your component, [follow the guide here](link here).
```

Reply:
Sure, now added the link.  Please see the screenshot from build-component page:
![screen shot 2018-11-27 at 9 36 56 pm](https://user-images.githubusercontent.com/5334504/49085718-9f328980-f28d-11e8-82ae-5868b5303698.png)